### DISCUSSION TITLE
Added Rowy Member, but it doesn't give him access as it should

### DISCUSSION THREAD

**Discussion by dvanduyne** [AntlerVC/firetable#1216] [Support / Q&A] [answered]
I've added a Member, and paid for the second seat (2 total, me and the new developer). He's now ADMIN, EDITOR, and OWNER.
[https://files.slack.com/files-pri/T02G2AYCUQP-F0533Q2A606/screenshot_2023-04-14_at_9.14.08_am.png](url)

I see all 3 tables in Rowy (I'm ADMIN and OWNER)
[https://files.slack.com/files-pri/T02G2AYCUQP-F0533Q16G5U/screenshot_2023-04-14_at_9.14.36_am.png](url)

But he only sees 1 of 3 tables
[https://files.slack.com/files-pri/T02G2AYCUQP-F0537DEPL13/screenshot_2023-04-14_at_12.11.58.png ](url)

Please advise

**Comment by BeeBombshell** [comment]
Hey @dvanduyne, I'm not able to access the screenshots you provided... Also, could you share your Project-Id and Email?

**Comment by harinij** [comment, marked as answer]
Hi @dvanduyne another thing you might want to check is if the 2 other tables have READ, WRITE access etc for the roles you have added - under the table settings?

![Screenshot 2023-04-20 at 5 14 38 pm](https://user-images.githubusercontent.com/307298/233289130-526b8bdf-b152-427a-835e-44c0ae22c8ee.png)
